flink-read-write-redis/README.md

```diff
@@ -0,0 +1,85 @@
+# Overview
+
+This example shows how to save input dataset with features into a Redis Cluster
+with `RedisSink`, and provides online serving based on the saved features with
+`RedisSource`. It involves the following steps:
+
+1. Read a batch of historical item price events from a file.
+
+   Each item price event has the following fields:
+   - item_id, unique identifier of the item.
+   - price, the new price of this item.
+   - timestamp, time when the new price is used for this item.
+
+2. For each item price event, save the event into a Redis cluster. If a feature
+   with the same item_id has been saved to Redis before, and the incoming event
+   has a larger timestamp than the existing one, Feathub would update the price
+   and timestamp values of the entry in Redis with that of the incoming event.
+
+3. Read the latest item prices from the Redis cluster and use them to create an
+   `OnDemandFeatureView` to provide online serving. In this example, the
+   features displayed by online serving are saved to a local file.
+
+# Prerequisites
+
+Prerequisites for running this example:
+- Unix-like operating system (e.g. Linux, Mac OS X)
+- Python 3.7
+
+# Step-By-Step Instructions
+
+Please execute the following commands under the `flink-read-write-redis` folder
+to run this example.
+
+1. Install Feathub and Flink pip package.
+
+   ```bash
+   $ python -m pip install apache-flink==1.15.2
```

Comment:
Given that FeatHub's setup.py has been updated to specify `apache-flink==1.15.2` in https://github.com/alibaba/feathub/pull/25, can we remove this comment from this example and other examples?